docs/data-guidelines.md

```diff
@@ -210,3 +211,27 @@ Valid support statements containing flags can be removed from BCD if it is consi
 These conditions represent minimum requirements for the removal of valid flag data; other considerations may result in flag data continuing to be relevant, even after the guideline conditions are met.
 
 This guideline was proposed in [#6670](https://github.com/mdn/browser-compat-data/pull/6670).
+
+## Initial versions for browsers
+
+The following table indicates initial versions for browsers in BCD. These are the earliest possible version numbers allowed to be used.
+
+For example, if the table indicates an initial version of "1" and an information source says a feature was implemented in a (beta) version "0.8", BCD requires you to record the intial version number "1" for it. In other words, a lower version number is always elevated to a browser's initial version number. Usually initial version numbers are "1" or "1.0"; the table below provides explanations when this is not the case.
+
+| Browser          | Initial version | Notes                                                                                                                    |
+| ---------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| Chrome           | 1               |                                                                                                                          |
+| Chrome Android   | 18              | First stable release using Chromium 18; older Android versions are ignored or listed under WebView Android.              |
+| Edge             | 12              | EdgeHTML, continuation of IE 11 (another initial version is Edge 79 when Edge switched to Chromium).                     |
+| Firefox          | 1               |                                                                                                                          |
+| Firefox Android  | 4               | ?!                                                                                                                       |
```

Comment:
I think version 4 was the first stable Firefox on Android. [It looks like there may have been some mobile versions of Firefox on Nokia devices](https://blog.mozilla.org/blog/2011/03/29/mozilla-launches-firefox-4-for-android-allowing-users-to-take-the-power-and-customization-of-firefox-everywhere-2/), but I don't think BCD is ignoring any releases on Android.